src/payment/payment-action-creator.ts

```diff
@@ -37,10 +37,11 @@ export default class PaymentActionCreator {
 
     initializeOffsitePayment(
         methodId: string,
-        gatewayId?: string
+        gatewayId?: string,
+        paymentData: PaymentInstrument = {}
```

Comment:
I believe this should just be `instrumentId` and `shouldSaveInstrument` as `PaymentInstrument` contains other fields such as card number and other sensitive data which are not applicable to an offsite payment.